google-cloud-bigquery/lib/google/cloud/bigquery/schema/field.rb

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
```

Comment:
We need two newlines or YARD will use the copyright header as documentation for the `Google` module. Can you restore the newline?